SonarQube.Common/AnalysisConfig/AnalysisConfig.cs

```diff
@@ -33,6 +33,12 @@ public class AnalysisConfig
         ///  See https://jira.sonarsource.com/browse/SONARMSBRU-100 for details.</remarks>
         public string SonarRunnerWorkingDirectory { get; set; }
```

Comment:
At this point I think we should also set the working dir to the sources dir (see  https://jira.sonarsource.com/browse/SONARMSBRU-100 for details)